Added instructions on how to generate PEM generate keypair, needed by JWT authentication (#2812)
ibexa/documentation-developer: docs/infrastructure_and_maintenance/security/development_security.md

                entry_point: lexik_jwt_authentication.jwt_token_authenticator
            stateless: true
```

In order for the application to be able to generate JWT tokens, a [PEM encoded keyset needs to be generated](https://symfony.com/bundles/LexikJWTAuthenticationBundle/current/index.html#generate-the-ssl-keys) using the command:

> 

Reviewer: To generate JWT tokens, you first need to generate a PEM encoded key pair.

```bash
php bin/console lexik:jwt:generate-keypair
```

The generated key pair will be stored in `config/jwt`

!!! note "[[= product_name_cloud =]]"

> 

Reviewer: To generate and store tokens on [[= product_name_cloud =]], you must define `config/jwt` as a volume in `.platform.app.yaml`. In a cluster setup, ensure the key pair is the same on all servers. You can achieve this by using a network share or by manually copying the key pair to all servers.

                entry_point: lexik_jwt_authentication.jwt_token_authenticator
            stateless: true
```

> In order for the application to be able to generate JWT tokens, a [PEM encoded keyset needs to be generated](https://symfony.com/bundles/LexikJWTAuthenticationBundle/current/index.html#generate-the-ssl-keys) using the command:

Reviewer: ```suggestion
Finish the setup by generating a [PEM encoded key pair](https://symfony.com/bundles/LexikJWTAuthenticationBundle/2.x/index.html#generate-the-ssl-keys) by using the command:
```
1) I'd use `key pair` consistently, imho no need to introduce "keyset" kere
2) Let's use the 2.x link for the 4.6 version (I will change it when upmerging)

```bash
php bin/console lexik:jwt:generate-keypair
```

> The generated key pair will be stored in `config/jwt`

Reviewer: ```suggestion
The generated key pair will be stored in the `config/jwt`directory.
```

!!! note "[[= product_name_cloud =]]"

>     In order to be able to store generate and store the tokens on [[= product_name_cloud =]], you must define `config/jwt`
>     as a volume in `.platform.app.yaml`. If you have a 3-node-cluster setup, you must ensure the key pair is the same on all
>     3 servers. Either use a network share for this, or use local mount manually copy the same keu pair
>     to all servers

Reviewer: ```suggestion
    To generate and store the tokens on [[= product_name_cloud =]], define the `config/jwt` directory as a volume in the `.platform.app.yaml` file. 
    In 3-node cluster setups, ensure that the key pair is the same on all 3 servers. 
    You can use a network share, or use a local mount and manually copy the key pair between the servers.
```
